application/src/main/java/org/opentripplanner/apis/gtfs/generated/GraphQLDataFetchers.java

```diff
@@ -408,6 +410,41 @@ public interface GraphQLFeedPublisher {
     public DataFetcher<String> url();
   }
 
+  /**
+   * Timing of an arrival or a departure to or from a stop. May contain real-time information if
+   * available. This is used when there is a known scheduled time.
+   */
+  public interface GraphQLFixedArrivalDepartureTime {
+    public DataFetcher<Object> estimated();
+
+    public DataFetcher<java.time.OffsetDateTime> scheduledTime();
+  }
+
+  /**
+   * Exact dated stoptime represents the time when a specific trip on a specific date arrives to and/or departs from a specific stop.
+   * This can include realtime estimates.
+   */
+  public interface GraphQLFixedStopTimeOnServiceDate {
+    public DataFetcher<FixedArrivalDepartureTime> arrival();
+
+    public DataFetcher<FixedArrivalDepartureTime> departure();
+
+    public DataFetcher<Object> stop();
+  }
+
+  /** A fixed (i.e. not flexible or frequency based) trip on a specific service date */
```

Comment:
I am a bit confused with the use of the terms "Fixed" and "Scheduled".
Are they synonymous in this context?

Reply:
Not really. We couldn't really figure out a really good name for this fixed. I think it was called exact at one point too. These "normal" trips have exact points of times for arrival and departure on stops (although real-time estimates can change this). Flex can have windows of time for arrival and departure and frequency based trips can have multiple departures in a day that repeat every n minutes. I think we consider both frequency and flex trips scheduled since they exist in the "plan data" (the static data, not originating from real-time updates).